src/include/distributed/placement_connection.h

```diff
@@ -23,7 +23,8 @@ extern MultiConnection * StartPlacementConnection(uint32 flags,
 												  const char *userName);
 
 extern void ResetPlacementConnectionManagement(void);
-extern void CheckForFailedPlacements(bool preCommit, bool using2PC);
+extern void MarkFailedShardPlacements(void);
+extern void PostCommitMarkFailedShardPlacements(bool using2PC);
```

Comment:
I'm not a fan of this.  It seems better to add a wrapper that calls CheckForFailedPlacements() as it is right now, with the adjusted parameters.